portality/templates/admin/edit_article_metadata.html

```diff
@@ -0,0 +1,161 @@
+{% extends "admin/admin_base.html" %}
+
+{% block content %}
+{% set object_type = 'article' %}
+{% set heading_object_type = 'Article' %}
+{% set form_context = form_context %}
+
+<div class="row" style="margin-bottom: 20px">
+    <div class="col-md-9">
+        <h1>{{ heading_object_type }}: {{form_context.source.bibjson().title}}</h1>
+        {% if form_context.source.last_updated %}
+          <h4>Last updated: {{form_context.source.last_updated}}</h4>
+        {% endif %}
+        {% if form_context.source.current_journal %}
+            <h4 style="margin-top: 1em;"><a href="{{ url_for('doaj.journal_readonly', journal_id=form_context.source.current_journal) }}" target="_blank">View associated journal</a></h4>
+        {% endif %}
+    </div>
+</div>
+
+<div class="row">
+<div class="col-md-12">
+
+    {% if there_were_errors %}
+        <h4 class="red">There is a problem with the submitted form.</h4>
+    {% endif %}
+
+    {% from "_formhelpers.html" import render_field_horizontal %}
+    {% from "_formhelpers.html" import render_field %}
+    <form method="post" action="{{ url_for('admin.article_page', article_id=form.id) }}" class="form-horizontal" id="article_metadata_form">
+
+        {% include "formcontext/maned_article_review.html" %}
```

Comment:
I'm not sure if this bit will change if we resurrect your initial FormContext approach, but probably we should give this a better name, since we wouldn't want to show the managing editor's form to the publisher (if they were different, which I realise they are not just now).  Probably call this `base_article_form.html` or something.